index.js

```diff
@@ -0,0 +1 @@
+console.log("ok")
```

Comment:
deveria exibir a mensagem "hello world"
```suggestion
console.log("hello world")
```